extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaImageInsertDialog.js

```diff
@@ -0,0 +1,230 @@
+/*!
```

Comment:
Files ve.ui.WikiaImageInsertDialog.js and ve.ui.WikiaVideoInsertDialog.js are huge copy & pastes which unfortunately makes this solution unacceptable. There are plenty of resource on the Internet explaining why code copy & paste is wrong but to name just few:
- from looking at this diff it's actually impossible to say what are the "slight" but important differences between those two files
- if we need to fix bugs - we need to do in two places (which is not common so it makes it more likely that one of the places gonna be missed)
- we lose history of the changes

Please consider what I was suggesting before, create new class that inherits from the already existing one and just overwrite some of the things (very common pattern in general and in VE specifically). I'm available for a meeting if you want to discuss it.